src/lightning/fabric/fabric.py

```diff
@@ -629,7 +633,7 @@ def load(
                 state[k] = unwrapped_state[k]
         return remainder
 
-    def launch(self, function: Optional[Callable[["Fabric"], Any]] = None, *args: Any, **kwargs: Any) -> Any:
+    def launch(self, function: Callable[["Fabric"], Any] = _do_nothing, *args: Any, **kwargs: Any) -> Any:
```

Comment:
This acts as a sentinel value to differentiate from `None` (no return of a function)